lib/plugins/filter/after_post_render/external_link.js

```diff
@@ -13,11 +13,11 @@ function externalLinkFilter(data) {
     return;
   }
 
-  data.content = data.content.replace(/<a\s+(?:[^<>]+\s)?href=["']([^<>"']+)["'][^<>]*>/gi, (str, href) => {
-    if (/target=/gi.test(str) || !isExternalLink(href, url, external_link.exclude)) return str;
+  data.content = data.content.replace(/<a(?:\s+?|\s+?[^<>]+)?href=["']([^<>"']+)["'][^<>]*>/gi, (str, href) => {
+    if (!isExternalLink(href, url, external_link.exclude) || /target=/i.test(str)) return str;
```

Comment:
Most renderer plugin won't add `target=`. By moving `isExternalLink` forward unnecessary regex matching could be avoided.